generators/client/index.js

```diff
@@ -401,6 +402,7 @@ module.exports = class extends BaseGenerator {
                     this.log(logMsg);
                 }
             );
+            codeFormat.formatter();
```

Comment:
don't do this instead do `this.spawnCommandSync(this.clientPackageManager, ['run', 'code-format'])` but call it before the build commands so may be at line 396/397 before invoking `webpack:build`